.travis.yml

```diff
@@ -10,6 +10,7 @@ before_install:
   - sudo apt-get install libtcmalloc-minimal4
   - export LD_PRELOAD="/usr/lib/libtcmalloc_minimal.so.4"
   - pip install --upgrade pip setuptools wheel
+  - pip install -U docutils==0.14
```

Comment:
Is this still needed? Or has the [July 21st `v0.15` release](https://pypi.org/project/docutils/0.15/) resolved this?

Reply:
Resolved in https://github.com/diana-hep/pyhf/pull/516/commits/faadc441dd43fb175aaebfe4cc4fff1ac995f611